Boa.Constrictor/Screenplay/Pattern/IConditionAdaptor.cs

```diff
@@ -0,0 +1,26 @@
+﻿namespace Boa.Constrictor.Screenplay
+{
+    /// <summary>
+    /// Adaptor interface for evaluating a condition.
+    /// </summary>
+    public interface IConditionAdaptor
```

Comment:
Note: The Adapter Pattern is typically written using "e" instead of "o":
https://en.wikipedia.org/wiki/Adapter_pattern